packages/lit-dev-content/site/tutorials/content/async-directive/02.md

```diff
@@ -0,0 +1,24 @@
+We've provided a sample element with a template. Go ahead and try importing your
+directive, and then we'll use it in a template.
+
+```ts
```

Comment:
Done. I'm putting them as comments in the snippets rather than markdown headers because the filenames themselves need to change based on js/ts. We should probably be consistent about how we do this across all tutorials, or else consider adding a "filename" treatment to the snippets themselves (in addition to fixing the file selection issue).

Reply:
Made PR to `playground-elements` to specify selected file in project.json: https://github.com/google/playground-elements/pull/304